docs/scenarios_authoring/Intro.md

```diff
@@ -46,36 +46,37 @@ SpEL [Spring Expression Language](https://docs.spring.io/spring-framework/docs/3
+The [SpEL Cheat Sheet page](Spel.md)  provides an exhaustive list of examples on how to write expressions with SpEL.
 
 &nbsp;
 ## Data Types
 
-Every SpEL expression returns a value of one of the predefined SpEL data types, like integer, double or boolean, map, etc. Data types in Nussknacker can be a confusing aspect at the beginning, as depending on the context in which data are processed or displayed, different data type schemes are in use - please refer to the [SpEL Cheat Sheet page](Spel.md#data-types-and-structures) for more information. 
+Every SpEL expression returns a value of one of the predefined SpEL data types, like integer, double, boolean, map, etc. Data types in Nussknacker can be a confusing aspect at the beginning, as depending on the context in which data is processed or displayed, different data type schemes will be used - please refer to the [SpEL Cheat Sheet page](Spel.md#data-types-and-structures) for more information. 
 
 In some contexts data type conversions may be necessary - conversion functions are described [here](Spel.md#type-conversions).
 
 &nbsp;
 ## Variables
 
-Nussknacker uses variables as containers for data. Variables have to be declared; a `variable` or `record-variable` component are used for this. Once declared, a hash sign `"#"` is used to refer to a variable from a SpEL expression. Variables are attributes of the data record, they do not exist by themselves. 
+Nussknacker uses variables as containers for data. Variables have to be declared; a `variable` or `record-variable` component is used for this. Once declared, a hash sign `"#"` is used to refer to a variable from a SpEL expression. Variables are part of a data record, they do not exist by themselves. 
 
 There are three predefined variables: `#input`, `#inputMeta` and `#meta`. 
 
-In the Streaming processing mode the `#input` variable is associated with the event which originally came from the Kafka topic. In the Request-Response processing mode the `#input` variable carries the request data of REST call which invoked Nussknacker scenario. Both in the Streaming and Request-Response cases some nodes not only terminate the input events, but also create new ones. As the result, the #input data record is no longer available after such a node, while the newly created data record (and the variable associated with it) is available "downstream". 
+In **Streaming** processing mode the `#input` variable is associated with an event that is provided by a Kafka topic. In the **Request-Response** processing mode the `#input` variable carries the request data of a REST call which invoked Nussknacker scenario. Both in the Streaming and Request-Response cases some nodes not only terminate input events but also create new ones. As the result, the `#input` data record is no longer available after such a node. The newly created data record (and the variable associated with it) is available "downstream" -in subsequent nodes. 
```

Comment:
if we do not like formulation _which originally came from the Kafka topic_, may  be _which was originally read from the Kafka topic_ would be better. I would never say that _event was provided by a Kafka topic_.